src/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs

```diff
@@ -548,4 +522,36 @@ public void SetIPProtectionLevel_ArgumentException(AddressFamily family)
             }
         }
     }
+
+    [Collection("NoParallelTests")]
+    // Set of tests to not run  together with any other tests.
+    public class NoParallelTests
```

Comment:
Perhaps this should be defined using `partial`?  Then the same test class name can be repeated in other test source files that require this type of execution (non-parallel) isolation.

Reply:
updated.